kit/Kit.cpp

```diff
@@ -1152,6 +1152,17 @@ class Document final : public DocumentManagerInterface
             }
             return;
         }
+        else if (type == LOK_CALLBACK_DOCUMENT_PASSWORD_RESET) {
```

Comment:
Please format using clang-format (easy to set vscode to apply it automatically for the modified lines when you save).